### DISCUSSION TITLE
Support cherry pick from one repo to another repo

### DISCUSSION THREAD

**Discussion by MatthieuMEZIL** [gitextensions/gitextensions#9881] [Q&A]
### Feature description

I have 2 different repositories. I was wondering if I could use gitextensions to cherry pick commits from one repo to another. 

### Environment

- GitExtensions version: 3.5.4.12724
- GIT version: 2.35.1.2
- OS version: Windows 10

**Comment by pmiossec** [comment]
Tiens, salut Mathieu (de quelqu'un qui t'a croisé sur une de tes missions 😉)

The feature request is not perfectly clear but this is not a feature provided by git so I don't think it would make sense to add it to gitextensions.

But you have 2 ways to achieve it.

You could use the format patch and then apply patch feature of GitExtensions (or git).

The other solution is to add the repo from where you want to take the commits as a remote inside the 2nd repository. Then fetch. You will be able to see the commits and be able to cherry-pick them. And gitextensions has also a feature to disable / enable the remote if you would like to do it again later.

**Comment by MatthieuMEZIL** [comment]
Sorry if it wasn't super clear. 
Actually, I think the only thing I need to do is to use: 
```
git remote add <repo2Name> <repo2Url> 
git fetch <repo2Name>
```
So sounds easy enough.

**Comment by pmiossec** [comment]
> Sorry if it wasn't super clear. Actually, I think the only thing I need to do is to use:
> 
> ```
> git remote add <repo2Name> <repo2Url> 
> git fetch <repo2Name>
> ```
> 
> So sounds easy enough.

These steps are done in the 'Remote repository' form by adding a new remote: https://git-extensions-documentation.readthedocs.io/en/master/remote_feature.html#manage-remote-repositories

And the URL provided could be one of a remote repository or a local path...
